.github/workflows/integ-tests.yml

```
@@ -0,0 +1,62 @@
+name: Integration Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  AWS_REGION : "us-west-2"
+  REGION : "us-west-2"
+  AMZN_BRAKET_OUT_S3_BUCKET : ${{ secrets.bucket_name }}
+  REPOSITORY_NAME : ${{ secrets.repository_name }}
+  ACCOUNT_ID : ${{ secrets.account_id }}
+  BUILD_RESULTS_PATH : "build_results.txt"
+
+permissions:
+  id-token: write   # This is required for requesting the JWT
+  contents: read    # This is required for actions/checkout
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+        container-type: ["base", "pytorch", "tensorflow"]
+    runs-on: ${{ matrix.os }}
+    env:
+      PREBUILD_TAG : ${{ matrix.container-type }}_prebuild_tag
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: configure aws credentials
+      uses: aws-actions/configure-aws-credentials@v4
```

```suggestion
      uses: aws-actions/configure-aws-credentials@e3dd6a429d7300a6a4c196c26e071d42e0343502
```

Taken from each package's release commit hash